frontend/src/components/ValidationErrors.tsx

```diff
@@ -10,57 +10,45 @@
 // limitations under the License.
 
 // UI Elements
-import { useTheme } from '@mui/material/styles';
+import { Icon } from "@awsui/components-react";
 
-import List from '@mui/material/List';
-import ListItem from '@mui/material/ListItem';
-import ListItemText from '@mui/material/ListItemText';
-
-// Icons
-import CheckIcon from '@mui/icons-material/Check';
-import CloseIcon from '@mui/icons-material/Close';
-
-export default function ValidationErrors(props: any) {
-  const theme = useTheme();
+export default function ValidationErrors({errors}: any) {
   const colorMap = (level: any) => {
     // @ts-expect-error TS(7053) FIXME: Element implicitly has an 'any' type because expre... Remove this comment to see the full error message
```

Comment:
It's not an easy one, but we could try and fix this by annotating `(level: string)`

The real fix would be typing the `errors` but that would involve changing how we store errors in the store, so it may not be feasible